### PR TITLE
Add Public Code Standard documentation/checks

### LINKED ISSUE #590: Make a Standard for Public Code analysis
**Is your feature request related to a problem? Please describe.**
As OpenZaak strives for being compliant with Standard for Public Code (with #565), it would be useful to see which requirements are met and which are not. 

**Describe the solution you'd like**
This issue aims to track the analysis of the codebase, and later the analysis result could be published as part of the documentation. And if not all requirements are met, separate issues could be created to address what is missing.

**Describe alternatives you've considered**
n/a

### PR DESCRIPTION
**Pull request by sergei-maertens**
Fixes #590

**Changes**

* Added markdown capabilities to Sphinx
* Added analysis result
* Included other markdown docs to fix the build

### CODE REVIEW

**Comment by sergei-maertens** [comment]
@ericherman I can't seem to add Jan as reviewer, do you have the github handle for me?

**Comment by ericherman** [comment]
To specifically add as a reviewer in github, perhaps @Ainali ( https://github.com/Ainali ) must be a member of the project?

File: docs/public_code.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+## [Create reusable and portable code](https://standard.publiccode.net/criteria/reusable-and-portable-codebases.html)
+
+- [x] designed to be reusable from the start
+
+Requirement | meets | links and notes
+-----|-----|-----
+The codebase MUST be developed to be reusable in different contexts. | yes | Designed to be so from the start.
+The codebase MUST be independent from any secret, undisclosed, proprietary or non-open licensed code or services for execution and understanding. | yes | installation supports docker, kubernetes, vmware appliances, bare-metal is possible
+The codebase MUST be in use by multiple parties. | yes | Deployed in multiple sandbox environments (e.g: Utrecht is testing it, others Den Haag, Delft looking at it.)
+The roadmap SHOULD be influenced by the needs of multiple parties. | yes | [Dimpact](https://www.dimpact.nl/openzaak), [market consultation](https://github.com/open-zaak/open-zaak-market-consultation) | Code SHOULD be general purpose and SHOULD be configurable. |
```
Comment (Ainali) [inline docs/public_code.md:37]: There is a missing line break here, making one requirement hidden.

Comment (sergei-maertens) [reply docs/public_code.md]: fixed

**Pull request merged**